bitnami/cassandra/Chart.yaml

```diff
@@ -22,4 +22,4 @@ maintainers:
 name: cassandra
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/cassandra
-version: 10.3.0
+version: 10.3.1
```

Comment:
As we are adding a new parameter to the chart, I think it would be more appropriate if it were a minor instead of a patch. So the version should be 10.4.0 instead of 10.3.1.

Reply:
Updated the version number.